pyproject.toml

```diff
@@ -2,9 +2,56 @@
 requires = [
     "setuptools>=46",
     "wheel",
-    "setuptools_scm",
-    "setuptools_scm_git_archive"
+    "setuptools_scm>=7",
+    "tomli>=1.0.0; python_version < '3.11'",
```

Comment:
What is this about?

Reply:
IDK actually. I have been cargo culting this around for a while now.